### PR TITLE
Troubleshooting: Update API

### PR DESCRIPTION
**Pull request by andyg0808**
The new Troubleshooting API returns an object with keys for each chunk of the page. This updates our rendering code to use that (and to render all the chunks which are available).

Closes ifixit/ifixit#46216

### CODE REVIEW

**Comment by andyg0808** [comment]
deploy_block :+1: on https://github.com/iFixit/ifixit/pull/46215

**Comment by deltuh-vee** [comment]
un_deploy_block 👍 
https://github.com/iFixit/ifixit/pull/46215 has shipped.

**Comment by andyg0808** [comment]
QA :+1: 
Page loads:
image

File: frontend/templates/troubleshooting/hooks/useTroubleshootingProps.tsx [hunk 1]
```diff
@@ -9,7 +9,9 @@ export type Problem = {
 export type TroubleshootingData = {
    title: string;
    toc: string;
-   groups: Problem[];
+   introduction: Problem[];
```
Comment (sctice-ifixit) [inline frontend/templates/troubleshooting/hooks/useTroubleshootingProps.tsx:12]: `Problem` is a misleading name now that we're using the same type for intro and conclusion sections. Are you intending to patch that up later?

**Review by sctice-ifixit — APPROVED** [review]
CR :flags: makes sense to me

deploy_block :cactus: on changing the name of `Problem` (nor or later), which doesn't seem to make sense. It seems more like a `Section`.

File: frontend/templates/troubleshooting/hooks/useTroubleshootingProps.tsx (hunk 1 above)
Comment (andyg0808) [reply frontend/templates/troubleshooting/hooks/useTroubleshootingProps.tsx:12]: Now I am. I think I'd patched it up in some other pull/commit, but I can't seem to find that one right now.

**Comment by andyg0808** [comment]
QA :+1: 
https://react-commerce-git-vulcan-update-wiki-api-ifixit.vercel.app/Vulcan/troubleshooting-test

image

**Comment by andyg0808** [comment]
un_deploy_block :+1:

File: frontend/templates/troubleshooting/hooks/useTroubleshootingProps.tsx (hunk 1 above)
Comment (andyg0808) [reply frontend/templates/troubleshooting/hooks/useTroubleshootingProps.tsx:12]: https://github.com/iFixit/react-commerce/pull/1305/commits/32a88d2bef4882a60f9a02c9b87c0771a600255e

**Pull request merged**